spread/client.go

```diff
@@ -296,7 +296,18 @@ func (c *Client) run(script string, dir string, env *Environment, mode outputMod
 		if err != nil {
 			return nil, err
 		}
-		c.Run("reboot", "", nil)
+
+		errch := make(chan error, 1)
+		go func() {
+			errch <- c.Run("reboot", "", nil)
+		}()
+
+		warn := time.NewTicker(c.warnTimeout)
+		select {
+		case <-errch:
+		case <-warn.C:
+			c.sshc.Close()
+		}
```

Comment:
here you're leaving the ticker running. This is probably not a huge problem, but it would be tidiest to do a
```go
warn.Stop()
```
in the `errch` case. On the other hand, if you don't think that's needed, you could've instead done directly
```go
case <- time.Tick(c.warnTimeout)
```